player/tests/data/quad.ron

```diff
@@ -0,0 +1,169 @@
+        CreatePipelineLayout(Id(0, 1, Empty), (
+            label: None,
+            bind_group_layouts: [],
+            push_constant_ranges: [],
+        )),
+        CreateRenderPipeline(Id(0, 1, Empty), (
+            label: None,
+            layout: Some(Id(0, 1, Empty)),
+            vertex_stage: (
+                module: Id(0, 1, Empty),
+                entry_point: "main",
+            ),
+            fragment_stage: Some((
+                module: Id(1, 1, Empty),
+                entry_point: "main",
+            )),
+            rasterization_state: None,
+            primitive_topology: TriangleList,
+            color_states: [
+                (
+                    format: Rgba8Unorm,
+                    alpha_blend: (
+                        src_factor: One,
+                        dst_factor: Zero,
+                        operation: Add,
+                    ),
+                    color_blend: (
+                        src_factor: One,
+                        dst_factor: Zero,
+                        operation: Add,
+                    ),
+                    write_mask: (
+                        bits: 15,
+                    ),
+                ),
+            ],
+            depth_stencil_state: None,
+            vertex_state: (
+                index_format: Uint16,
+                vertex_buffers: [],
+            ),
+            sample_count: 1,
+            sample_mask: 4294967295,
+            alpha_to_coverage_enabled: false,
+        )),
+        Submit(1, [
+            RunRenderPass(
+                base: (
+                    commands: [
+                        SetPipeline(Id(0, 1, Empty)),
+                        Draw(
+                            vertex_count: 6,
+                            instance_count: 1,
+                            first_vertex: 0,
+                            first_instance: 0,
+                        ),
+                    ],
+                    dynamic_offsets: [],
+                    string_data: [],
+                    push_constant_data: [],
+                ),
+                target_colors: [
+                    (
+                        attachment: Id(0, 1, Empty),
+                        resolve_target: None,
+                        channel: (
+                            load_op: Clear,
+                            store_op: Store,
+                            clear_value: (
+                                r: 0,
+                                g: 0,
+                                b: 0,
+                                a: 1,
+                            ),
+                            read_only: false,
+                        ),
+                    ),
+                ],
+                target_depth_stencil: None,
+            ),
+            CopyTextureToBuffer(
+                src: (
+                    texture: Id(0, 1, Empty),
+                    mip_level: 0,
+                    array_layer: 0,
+                    origin: (
```

Comment:
I'm thinking that we should sprinkle `serde(default)` on all those fields with default values, so that writing tests is more egonomic

Reply:
So should the serde defaults be enabled for tracing, or do you want to leave them just for replays?

Reply:
let's leave them for replays only, so that we don't overburden the compile times when only the tracing feature is on